src/apply_manual_codes.py

```diff
@@ -0,0 +1,140 @@
+import time
+from os import path
+import sys
+
+from core_data_modules.cleaners import Codes
+from core_data_modules.cleaners.cleaning_utils import CleaningUtils
+from core_data_modules.traced_data import Metadata
+from core_data_modules.traced_data.io import TracedDataCodaV2IO
+from core_data_modules.util import TimeUtils
+
+from src.lib import PipelineConfiguration, CodeSchemes
+
+class ApplyManualCodes(object):
+    @classmethod
+    def apply_manual_codes(cls, user, data, coda_input_dir):
+        # Merge manually coded radio show files into the cleaned dataset
+        for plan in PipelineConfiguration.RQA_FOLLOW_UP_CODING_PLANS:
```

Comment:
This was changed to RQA + FOLLOW_UP in a future PR right?

If not, don't merge this.

Reply:
Erroneously committed here. Fixed in future pr